src/pages/host/getHostEventString.tsx

```diff
@@ -290,6 +290,22 @@ export const getHostEventString = (
           Expiration warning sent
         </span>
       );
+    case HostEvent.VolumeMigrationFailed:
+      return (
+        <span data-cy="host-volume-migration-failed">
+          Home volume failed to migrate to new host.
+          {data.logs ? (
+            <HostEventLog
+              title="Volume migration logs"
+              logs={data.logs}
+              isCode
+            />
+          ) : (
+            ""
+          )}
```

Comment:
I think you can do this!
```suggestion
          {data.logs && (
            <HostEventLog
              title="Volume migration logs"
              logs={data.logs}
              isCode
            />
          )}
```